resources/acsl/assembly.md

```diff
@@ -149,7 +149,7 @@ Look at the following table below to understand how the program changes over tim
 | MULT | 8 | 1 | 4 |
 | STORE | 8 | 1 | 8 |
 | LOAD | 1 | 1 | 8 |
-| SUB | 0 | 1 | 8 |
+| SUB | `$0$` | 1 | 8 |
```

Comment:
If you're going to make this LaTeX make every other number LaTeX too

Reply:
Not sure why you want to use LaTeX though. Just stick to regular numbers in my opinion

(So convert everything back to just numbers maybe?)